--- configure.ac	2012-09-30 10:53:32.000000000 -0700
+++ configure.ac	2014-07-08 18:02:20.285143803 -0700
@@ -41,9 +41,12 @@
 CPPUNIT_CFLAGS="$CPPUNIT_CFLAGS -DZKSERVER_CMD=\"\\\"${base_dir}/src/c/tests/zkServer.sh\\\"\""
 else
 CPPUNIT_CFLAGS="$CPPUNIT_CFLAGS -DZKSERVER_CMD=\"\\\"./tests/zkServer.sh\\\"\""
-AC_CHECK_FILES([generated/zookeeper.jute.c generated/zookeeper.jute.h],[],
-    [AC_MSG_ERROR([jute files are missing! Please run "ant compile_jute" while in the zookeeper top level directory.])
-])
+if test -f "generated/zookeeper.jute.c"; then :; else
+   AC_MSG_ERROR([jute files are missing! Please run "ant compile_jute" while in the zookeeper top level directory.])
+fi
+if test -f "generated/zookeeper.jute.h"; then :; else
+   AC_MSG_ERROR([jute files are missing! Please run "ant compile_jute" while in the zookeeper top level directory.])
+fi
 fi
 AC_SUBST(CPPUNIT_CFLAGS)
 
