--- mt_adaptor.c	2012-09-30 10:53:32.000000000 -0700
+++ mt_adaptor.c	2014-07-01 16:05:07.387967525 -0700
@@ -482,6 +482,9 @@
 
 int32_t fetch_and_add(volatile int32_t* operand, int incr)
 {
+#if defined(__GNUC__)
+  return __sync_fetch_and_add(operand, incr);
+#else
 #ifndef WIN32
     int32_t result;
     asm __volatile__(
@@ -503,6 +506,7 @@
      }
      return result;
 #endif
+#endif
 }
 
 // make sure the static xid is initialized before any threads started
